blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
```

Comment:
@aduth thoughts on a separate default method for this? I think it would clean up edit implementation a bit.